fecgov/fec-cms
# Feature/4220 node package vulnerabilities

## Summary

- Resolves #4220 

Updating packages that `npm audit fix` could fix with only a patch-level fix.
One of those required an update to the polyfills file.

## Reviewers
Feel free to add or remove—not sure exactly who would _need_ to approve

## Impacted areas of the application

Mostly the build and automated tests but, since JavaScript is involved, it could affect any of the interactive parts of our site, too (datatables, maps, widgets, admin, etc)

## Screenshots

None

## Related PRs

None

## How to test

- Pull the branch like normal
- `npm i` to install updated packages. It should end with at note about (at current) 23 low severity vulnerabilities
- `npm audit` will give you more details if you're interested
- `npm run build` will build and minify the CSS and JavaScript
  - (`npm run build-sass` will build the css
  - `npm run build-js` will build the JavaScript
  - `npm run build` will do both)
- `npm run test-single` will run the tests
- `./manage.py runserver`
- Check that [localhost](http://127.0.0.1:8000/] is working as expected, including
  - homepage
  - datatables pages (including sorting)
  - maps (like on the homepage and 
  - widgets (Aggregate Totals, Where Contributions Come From, Presidential Financial Map, Admin Fines Calculator)



____
